API.md

```diff
@@ -464,7 +520,9 @@ object about the version retrieved.
 
 | Property         | Type      | Description
 | ---              | ---       | ---
-| @context         | String    | The RERUM context file http://store.rerum.io/v1/context.json
+| @context         | String    | The RERUM context file http://devstore.rerum.io/v1/context.json.

```

Comment:
This should be store, not devstore, since this is the general API

Reply:
This is one of the few spots I am ok with using the prod namespace.  I will make that change.